test/test_config.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+
+import pytest
```

Comment:
![screen shot 2017-07-03 at 2 35 37 pm](https://user-images.githubusercontent.com/5179553/27804521-581e75e6-5ffd-11e7-9989-304c303fed6e.png)
Looks way better than unittest in use and also is very easy to fold in codecov integration, which is my next step.